packages/snaps-utils/src/index.executionenv.ts

```diff
@@ -6,3 +6,4 @@ export * from './iframe';
 export * from './logging';
 export * from './namespace';
 export * from './types';
+export * from './endowments';
```

Comment:
```suggestion
```

I don't think this is used in execution environments? If it is we can leave it, but otherwise we don't need to add it